manifests/sentinel_params.pp

```diff
@@ -32,7 +32,7 @@
       $conf_logrotate = '/etc/logrotate.d/redis-sentinel'
       $pidfile        = '/var/run/redis/redis-sentinel.pid'
       $logfile        = '/var/log/redis/redis-sentinel.log'
-      $upstart_script = '/etc/init/redis-sentinel.conf'
+      $upstart_script = '/etc/init.d/redis-sentinel'
```

Comment:
Let's rename the variable to $init_script since we're not using upstart anymore to be less confusing.